pretty/src/pane.rs

```diff
@@ -104,8 +82,9 @@ pub enum PaneError {
     #[error("invalid pane notation")]
     InvalidNotation,
 
+    // TODO use Display representation, not debug
```

Comment:
TODO: require both Debug + Display, use Display for printing MissingLabel error.